### PR TITLE
Ar 601 audio blog post tab

### PR DESCRIPTION
**Pull request by wissog**
i see ido did his backend part as well so language filter already works there, so you can merge

### CODE REVIEW

**Comment by wissog** [comment]
It worked on Friday with Ido's changes

It is simple, you don't need me to merge it

**Comment by gshilin** [comment]
I need you

File: src/components/Sections/Programs/MainPage.js [hunk 1]
```diff
@@ -27,19 +27,22 @@ class MainPage extends PureComponent {
     resetNamespace: PropTypes.func.isRequired
   };
 
-  componentWillReceiveProps(nextProps) {
-    const { match, location } = this.props;
-    const tab                 = match.params.tab || tabs[0];
-    const nextTab             = nextProps.match.params.tab || tabs[0];
+  componentDidMount(prevProps){
```
Comment (gshilin) [inline src/components/Sections/Programs/MainPage.js]: change to componintDidUpdate()

File: src/components/Sections/Programs/MainPage.js [hunk 2]
```diff
@@ -59,28 +62,26 @@ class MainPage extends PureComponent {
...
       <div>
         <SectionHeader section="programs" submenuItems={submenuItems} />
-        {content}
+        {content(active)}
       </div>
     );
   }
 }
 
+const content = (active) => {
```
Comment (gshilin) [inline src/components/Sections/Programs/MainPage.js]: make it static function of the above class

File: src/components/Sections/Publications/MainPage.js [hunk 3]
```diff
@@ -60,31 +64,30 @@ class MainPage extends PureComponent {
...
       <div>
         <SectionHeader section="publications" submenuItems={submenuItems} />
-        {content}
+        {content(active)}
       </div>
     );
   }
 }
 
+const content = (active) => {
```
Comment (gshilin) [inline src/components/Sections/Publications/MainPage.js]: make it static function of the above class

File: src/components/Sections/Publications/tabs/AudioBlog/Container.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            )
+            : null
+        }
+      </Table.Cell>
+    </Table.Row>
+  );
+};
+
+
+class AudioBlogList extends PureComponent {
```
Comment (gshilin) [inline src/components/Sections/Publications/tabs/AudioBlog/Container.js]: make if functional component

**Comment by wissog** [comment]
merge it @gshilin

**Pull request merged**